Index: f1
===================================================================
--- f1  (.../branches/b1)       (revision 14)
+++ f1  (.../trunk)     (revision 14)
@@ -1 +1,4 @@
 line1
+lnie2
+
+aaa
Index: file.c
===================================================================
--- file.c      (.../branches/b1)       (nonexistent)
+++ file.c      (.../trunk)     (revision 14)
@@ -0,0 +1 @@
+void aa(){}
